contrib/rabbitmq/samples/README.md

```diff
@@ -0,0 +1,141 @@
+# Rabbitmq - Source
+The Rabbitmq Event source enables Knative Eventing integration with Rabbitmq. When an message is produced to Rabbitmq, the
+Rabbitmq Event Source will consume the produced message and post that message the corresponding event sink.
+  
+This sample demonstrates how to configure, deploy, and use the Rabbitmq Event Source with a Knative Service.
+
+## Build and Deploy Steps
+### Prerequisites
+1. An existing instance of Rabbitmq must be running to use the Rabbitmq Event Source.
+    - The broker corresponding to Rabbitmq instance must be obtained.
+2. Install the `ko` CLI for building and deploying purposes.
+    ```
+    go get github.com/google/go-containerregistry/cmd/ko
+    ```
+3. A container registry, such as a Docker Hub account, is required.
+    - Export the `KO_DOCKER_REPO` environment variable with a value denoting the container registry to use.
+      ```
+      export KO_DOCKER_REPO="docker.io/YOUR_REPO"
```

Comment:
I think you have to add you username/YOUR_REPO now in docker hub

Reply:
Hi @Justin2997, thanks for your comment. By `docker.io/YOUR_REPO` I meant, `docker.io/<username>`.  Please let me know if you would like `YOUR_REPO` to be changed.